sky/cli.py

```diff
@@ -1066,6 +1118,14 @@ def _make_task_or_dag_from_entrypoint_with_overrides(
     disk_size: Optional[int] = None,
     disk_tier: Optional[str] = None,
     ports: Optional[Tuple[str]] = None,
+    readiness_path: Optional[str] = None,
+    initial_delay_seconds: Optional[int] = None,
+    min_replicas: Optional[int] = None,
+    max_replicas: Optional[int] = None,
+    target_qps_per_replica: Optional[float] = None,
+    post_data: Optional[str] = None,
+    upscale_delay_seconds: Optional[int] = None,
+    downscale_delay_seconds: Optional[int] = None,
```

Comment:
I am not exactly sure about having these service related arguments in the CLI as they might be a bit too detailed for a service. Can we leave them out for now (e.g., keep them in another branch), and only merge the CLI arguments for resources for `sky serve` CLI?

Reply:
Good point! Done in #2979 and PTAL